devtools/maven/src/main/java/io/quarkus/maven/NativeImageMojo.java

```diff
@@ -36,6 +36,14 @@
 import io.quarkus.creator.phase.nativeimage.NativeImagePhase;
 import io.quarkus.creator.phase.runnerjar.RunnerJarOutcome;
 
+/**
+ * Build a native executable of your application.
+ * It improves the startup time of the application, and produces a minimal disk footprint.
+ * The executable would have everything to run the application including the "JVM" (shrunk to be just enough to run the
+ * application), and the application.
+ * <p>
+ * You can build the application before with {@code build}
```

Comment:
I think you should just keep `Build a native executable of your application.`. The rest is too verbose and is not really useful IMHO.